### PR TITLE
Replace some hard-coded strings with localized

### PR DESCRIPTION
**Pull request by m4tx**

### CODE REVIEW

**Comment by nemobis** [comment]
Ping @benapetr, cc @siebrand

File: huggle/Localization/en.xml [hunk 1]
```diff
@@ -552,9 +563,13 @@
     <string name="warning-warntype">Warning type:</string>
     <string name="warning-warnlog">Warnings for this user:</string>
     <string name="warning-submit">Send</string>
+    <string name="wikiedit-tp-fail">Unable to retrieve $1 warning level will not be scored by it</string>
     <string name="missing-warning">There is no such warning template $1</string>
     <string name="han-not">Vandalism network isn't allowed in options</string>
     <string name="han-connecting">Connecting to Huggle anti vandalism network</string>
+    <string name="han-already-connected">Not connecting to HAN because it's already connected</string>
+    <string name="han-already-connecting">Please wait, I am already connecting to HAN</string>
+    <string name="han-disconnected">You are disconnected from HAN</string>
```
Comment (nemobis) [inline huggle/Localization/en.xml]: Are not connected? Were disconnected?

File: huggle/Localization/en.xml [hunk 2]
```diff
@@ -394,6 +399,10 @@
     <string name="main-revision-acceptpend">Accept this pending revision</string>
     <string name="main-no-page">The action you have requested requires a page to be loaded. Please load a page before trying again.</string>
     <string name="main-revert-null">Unable to revert: edit is null</string>
+    <string name="main-revert-newpage-title">Can't revert this</string>
+    <string name="main-revert-newpage">This is a new page, so it can't be reverted, you can either tag it, or delete it.</string>
+    <string name="main-restore-text-fail">Unable to restore the revision, because there is no text available for it</string>
+    <string name="main-restore-data-fail">Unable to restore the revision because wiki provided no data for selected version</string>
```
Comment (nemobis) [inline huggle/Localization/en.xml]: Comma before because? "the wiki"?

File: huggle/Localization/qqq.xml [hunk 3]
```diff
@@ -314,6 +315,8 @@ The title for this error is {{msg-huggle|Error}}.</string>
 {{Identical|IRC}}</string>
   <string name="main-system-change-provider-wiki">{{Identical|Wiki}}</string>
   <string name="main-system-exit">{{Identical|Exit}}</string>
+  <string name="main-config-state-fail">Used as error log entry.</string>
+  <string name="main-config-geom-fail">Used as error log entry.</string>
```
Comment (nemobis) [inline huggle/Localization/qqq.xml]: Would be useful to link or explain what "geometry" means in this context

File: huggle/Localization/en.xml [hunk 4]
```diff
@@ -361,12 +364,14 @@
     <string name="main-user-report">Report</string>
     <string name="main-user-ib">Increase badness score</string>
     <string name="main-user-clear-tp">Clear talk page of user</string>
+    <string name="main-user-retrieving-tp">Retrieving TP of %1</string>
```
Comment (nemobis) [inline huggle/Localization/en.xml]: Is TP really a common shortening in this product? It probably shouldn't ."Talk" is just two letters more and translators can decide whether to shorten or not. If needed for consistency with other messages, ok for this patch but please file an issue.

**Comment by m4tx** [comment]
@nemobis I fixed the issues you mentioned.

**Pull request merged**